dissect/target/plugins/general/users.py

```diff
@@ -1,10 +1,13 @@
+from functools import lru_cache
 from typing import Generator, NamedTuple, Optional, Union
 
 from dissect.target.exceptions import UnsupportedPluginError
 from dissect.target.helpers.fsutil import TargetPath
 from dissect.target.helpers.record import UnixUserRecord, WindowsUserRecord
 from dissect.target.plugin import InternalPlugin
 
+from ... import Target
```

Comment:
Can you make this an absolute import?

Reply:
Oh, I missed that one. Unwanted side effects from letting your IDE create the init ;)